### PR TITLE
[Cleanup][B-20] Replace `to_variable` with `to_tensor`

### PR DESCRIPTION
**Pull request by co63oc**
### PR types

Others 
### PR changes

Others 
### Description

https://github.com/PaddlePaddle/Paddle/issues/61385

### CODE REVIEW

File: python/paddle/nn/layer/layers.py [hunk 1]
```diff
@@ -167,9 +167,9 @@ def _multiple_input(self, inputs_in):
         ret = []
         if isinstance(inputs, (list, tuple)):
             for inp in inputs:
-                ret.append(self.to_variable(inp))
+                ret.append(paddle.to_tensor(inp))
```
Comment (SigureMo) [inline python/paddle/nn/layer/layers.py]:
```suggestion
                ret.append(self.to_variable(inp))
```

File: python/paddle/nn/layer/layers.py [hunk 2]
```diff
@@ -167,9 +167,9 @@ def _multiple_input(self, inputs_in):
...
         else:
-            ret.append(self.to_variable(inputs))
+            ret.append(paddle.to_tensor(inputs))
```
Comment (SigureMo) [inline python/paddle/nn/layer/layers.py]:
```suggestion
            ret.append(self.to_variable(inputs))
```

**Review by sunzhongkai588 — APPROVED** [review]
LGTM～

**Review by SigureMo — APPROVED** [review]
LGTMeow

**Pull request merged**